eav/queryset.py

```diff
@@ -282,3 +282,79 @@ def get(self, *args, **kwargs):
         the ``Manager`` get method.
         """
         return super(EavQuerySet, self).get(*args, **kwargs)
+
+
+    def order_by(self, *fields):
+        """
+        Django doesn't support arbitrary order-by clauses.
```

Comment:
I would love to see a little more verbose description of what is an "arbitrary order-by clause". After reading the docstring, if I didn't know the problem solved by this PR, I would have little idea of the purpose of this method.